source/frontend/src/features/leases/add/AddLeaseYupSchema.ts

```diff
@@ -114,6 +114,11 @@ export const AddLeaseYupSchema = Yup.object().shape({
   primaryArbitrationCity: Yup.string()
     .nullable()
     .max(200, 'Primary arbitration city must be at most ${max} characters'),
+  isPublicBenefit: Yup.string().nullable(),
```

Comment:
As far as I can tell, these are defined as booleans in the frontend model.

Reply:
Taken out the nullable validation on the yup schema since is not needed.